nit: Only add screenshot / screencast if PR change is UI related

cdrage opened the pull request:
nit: Only add screenshot / screencast if PR change is UI related

### What does this PR do?

Updates the PR template to indicate that we should only upload a
screenshot / screencast if it is UI related, as code-related changes
cannot be shown graphically.

### Screenshot/screencast of this PR

<!-- If code-replated, include a screenshot or
a screencast explaining this PR -->

N/A

### What issues does this PR fix or reference?

<!-- Include any related issues from Podman Desktop
repository (or from another issue tracker). -->

N/A

### How to test this PR?

<!-- Explain steps to reproduce -->

N/A

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

deboer-tim reviewed (commented):
No more pleases? ;-)

The wording is throwing me off a bit, maybe this?
```
<!-- If this PR is changing UI, include screenshots
or screencasts showing the difference -->
```
Also hints that there could be multiple or before/after without requiring it.

cdrage commented:
> No more pleases? ;-)
> 
> The wording is throwing me off a bit, maybe this?
> 
> ```
> <!-- If this PR is changing UI, include screenshots
> or screencasts showing the difference -->
> ```
> 
> Also hints that there could be multiple or before/after without requiring it.

Updated and added more Canadian niceness to the template 🇨🇦

deboer-tim reviewed (approved):
LGTM
